incubator/swift/image/project/Dockerfile

```diff
@@ -19,7 +20,8 @@ FROM swift:5.0-slim
 
 # Install OS updates
 RUN apt-get update \
- && apt-get install -y libcurl4-openssl-dev libssl-dev \
+ && apt-get dist-upgrade -y \
+ && apt-get install -y zlib1g-dev libcurl4-openssl-dev libssl-dev \
```

Comment:
I think we just need `zlib1g` not the dev package as this is the runtime slim section?  Should only need the dev headers for the build.

Reply:
Also, `libcurl4` and `libssl1.1`, maybe (haven't tested it).

Reply:
Also, while you're in here can we change `swift build -c release > output.txt` to `swift build -c release | tee output.txt` so the output goes to both stdout and the file. Otherwise the user can't see their app being built.

Reply:
Maybe we can even get rid of the `output.txt` and do something like `swift build -c release | tee awk 'END {print $NF}' >> run.sh` - dunno.

Reply:
I've added `tee`, but left it as two separate tasks.